server/models/transaction.go

```diff
@@ -6,6 +6,7 @@ type Transaction struct {
 	TxHash          string    `json:"tx_hash" bson:"tx_hash"`
 	To              string    `json:"to" bson:"to"`
 	From            string    `json:"from" bson:"from"`
+	Status          bool      `json:"status" bson:"status"`
```

Comment:
What about `Success` instead?

Reply:
let's leave it like to make it more similar to web3 tx

Reply:
I'm not sure what you mean, but how about leaving a comment like this?
```suggestion
	Status          bool      `json:"status" bson:"status"` // True: Successful, False: Failed
```